src/workflows-app/components/TroubleshootingBox.ts

```diff
@@ -0,0 +1,99 @@
+import { div, h, span } from 'react-hyperscript-helpers';
+import { ClipboardButton } from 'src/components/ClipboardButton';
+import { Link } from 'src/components/common';
+import { icon } from 'src/components/icons';
+import colors from 'src/libs/colors';
+import * as Nav from 'src/libs/nav';
+import { LogTooltips } from 'src/workflows-app/utils/task-log-utils';
+
+type TroubleshootingBoxProps = {
+  name: string;
+  namespace: string;
+  logUri: string;
+  submissionId: string;
+  workflowId: string;
+  showLogModal: (modalTitle: string, logsArray: any, tesLog: string) => void;
```

Comment:
Generally, suggest considering an options object when a function has several arguments.
```suggestion
  showLogModal: (args: { modalTitle: string; logsArray: any[]; tesLog: string; }) => void;
```
https://www.codereadability.com/what-are-javascript-options-objects/

Also, nit on the `logsArray` type: it sounds like it can at least by typed as an array?